test/format/config_def.c

```diff
@@ -129,6 +129,12 @@ CONFIG configuration_list[] = {
   {"disk.mmap_all", "configure mmap operations (read and write)",
     C_BOOL, 5, 0, 0, V_GLOBAL_DISK_MMAP_ALL},
 
+  {"file_manager.close_handle_minimum", "number of handles open before the file manager will look for handles to close",
+    0x0, 0, 50, 100, V_GLOBAL_FILE_MANAGER_CLOSE_HANDLE_MINIMUM},
```

Comment:
```
Config('file_manager', '', r'''
        control how file handles are managed''',
        type='category', subconfig=[
        Config('close_handle_minimum', '250', r'''
            number of handles open before the file manager will look for handles to close''',
            min=0),
        Config('close_idle_time', '30', r'''
            amount of time in seconds a file handle needs to be idle before attempting to close
            it. A setting of 0 means that idle handles are not closed''',
            min=0, max=100000),
        Config('close_scan_interval', '10', r'''
            interval in seconds at which to check for files that are inactive and close them''',
            min=1, max=100000),
        ]),
```
Here's the config for WiredTiger. I can see close_handle_minimum doesn't have a maximum value. Given that we only can run maximum 10 tables in test format, we should set the maximum to 10? Can you also add a comment to explain why a maximum is needed here?

Reply:
Do we also need to add config for `close_scan_interval`?

Reply:
Thanks! I have updated this to be 10 and added a comment

Reply:
On the ticket it says:

`Test/format can only open up to 32 tables at once`

I think we need to change the max to 32.

Reply:
Can you also run a stress test with the new config?

Reply:
The ticket doesn't specify to add a config for close_scan_interval, we can ask @jiechenbo about this.
I will update the max to be 32 and have run a patch with stress testing for test/format.

Reply:
I would prefer that we add a config for close_scan_interval, thank you for enquiring @sonar3ddy

Reply:
I have pushed the changes with the new config :)

Reply:
I have pushed this change and started a patch build [here](https://spruce.mongodb.com/version/63a3b95d5623432f2e0cfd46/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).